eng/pipelines/templates/jobs/aggregate-dependencies.yml

```diff
@@ -0,0 +1,40 @@
+
+jobs:
+- job: 'ValidateDependencies'
+
+  pool:
+    vmImage: 'windows-2019'
```

Comment:
We should use image `ubuntu-18.04` by default, and only use `windows-2019` when Windows is a requirement.  Is Windows required for this pipeline?

Reply:
Right. Tje Azure CopyFiles Task (the one we are using for uploading to blobs) requires windows

Reply:
Yeah so I upgraded to AzureFileCopy@3 from version 2. But that one also supports only the windows based powershell. I think they haven't designed this task for powershell core unfortunately.